### DISCUSSION TITLE
How can I sponsor this project?

### DISCUSSION THREAD

**Discussion by lavrov** [outwatch/outwatch#612] [General]
It has been a long time since I looked at outwatch last time. The project has changed a lot. There is new documentation, many effect libraries are supported out of the box etc. You have put a lot of effort in it. I would be happy to support this project with my modest contibution.

**Comment by fdietze** [comment]
Thank you for appreciating the effort!
We haven't thought about sponsoring yet, so far. Do you have any suggestions? How are other projects dealing with sponsoring?

**Reply by lavrov** [reply]
I find https://github.com/sponsors very convenient.

**Reply by cornerman** [reply]
Thank you for the kind words :)
Does github sponsors work for a team of developers (organization) as well?
